importer/import.py

```diff
@@ -20,11 +20,12 @@ def retry_if_db_error(exception):
 # we wait a bit for postgres to be ready since if used with docker, so docker might take a while to start
 @retry(stop_max_delay=10000, retry_on_exception=retry_if_db_error)
 def _pg_connect():
+    host = environ.get("POSTGRES_HOST")
```

Comment:
for this to be retrocompatible (but should we?), maybe you can add a default value `postgres` ?

Reply:
Yes, I wasn't sure about this one. The readme indicates that docker and docker-compose are a prerequisit, and the `.env` file is stated in the docker-compose...
I don't think it's necessary, but if you insists I can add it. What do you think ?

Reply:
Yes the `POSTGRES_HOST` is already present in the default `.env` and used by the API, so it should be ok.

Reply:
top!